pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -297,6 +297,13 @@ public void start() throws Exception {
                 "c-" + config.getClusterName()
                     + "-fw-" + hostname
                     + "-" + workerConfig.getWorkerPort());
+            // inherit broker authorization setting
```

Comment:
same question as above regarding inheriting the zookeeper server settings.

Reply:
I replied in your previous comment